### PR TITLE
Initial contribution

### PR DESCRIPTION
**Pull request by snovikov**

### CODE REVIEW

**Review by cytopia — APPROVED** [review]

**Review by titanlien — APPROVED** [review]
LGTM, but do we have use case for this module ?

File: README.md [hunk 1]
```diff
@@ -26,13 +25,26 @@ No providers.
...
 
 <!-- TFDOCS_REQUIREMENTS_END -->
 
 <!-- TFDOCS_INPUTS_START -->
 ## Required Inputs
 
-No required inputs.
+The following input variables are required:
+
+### <a name="input_token"></a> [token](#input\_token)
```
Comment (danvaida) [inline README.md:37]: I don't like the seemingly standard "input_" prefix for every variable.
It's probably coming from the docs generator but I don't subscribe to this standard.
Taking this one notch further, we can also rename the variable to actually have its name a bit more suggestive.

File: README.md [hunk 2]
```diff
@@ -26,13 +25,26 @@ No providers.
...
+
+Description: Github token to be used for given organization to manage its settings.
+
+Type: `string`
+
+### <a name="input_owner"></a> [owner](#input\_owner)
```
Comment (danvaida) [inline README.md:43]: Same thing as with the above.

**Comment by danvaida** [comment]
> LGTM, but do we have use case for this module ?

IIRC, we plan to manage our GH org with this module...

File: README.md (hunk 1 above)
Comment (snovikov) [reply README.md:37]: @danvaida this is just an reference which terraform-docs generate. check variable definition - it is `token`

File: README.md (hunk 2 above)
Comment (snovikov) [reply README.md:43]: the same

File: README.md (hunk 1 above)
Comment (danvaida) [reply README.md:37]: I know. I just like to have my documentation matching the name of the vars and outputs one-to-one, w/o any other decorations.
But that's just me...

**Pull request merged**